.gitignore

```diff
@@ -73,6 +73,9 @@ _deps
 # Compiled artifacts
 azure-nvme-id
 build/
-udev/80-azure-nvme.rules
-src/version.h
+debian/
```

Comment:
This should be `/debian`, or else you'll end up ignoring `packaging/debian` as well, which is not what you want.

The line above it should probably also be `/build`.

Reply:
fixed, thanks!